### PR TITLE
Added history parsing

### PR DESCRIPTION
**Pull request by stepbystepdayo**
I imported os and path then added history parsing. also I updated README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,25 @@
 # terminal-history
 
-**Get the command history of the user's shell**
+> Get the command history of the user's shell\*\*
```
Comment (tryingtokeepup) [inline README.md]: Remove the last three chars here, please.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
 ```
 
-## Reference (WRITE GOOD CODE!)
+## Note:
+
+Currently, this doesn't work for Windows. We will update this in the future.
+
+## Reference
+
+[shell-history](https://github.com/sindresorhus/shell-history) by [sindresorhus](https://github.com/sindresorhus).
```
Comment (tryingtokeepup) [inline README.md:23]: Good job, let's always show who we are borrowing ideas from!

File: index.js [hunk 3]
```diff
@@ -8,9 +10,32 @@ let numberToShow = 0;
 // Read the history file and parse the data
 
 function readFile() {
-  const array = fs.readFileSync("testHistory.txt").toString().split("\n");
+  const homeDir = os.homedir();
+  const historyPath = path.join(homeDir, ".bash_history");
+  const possiblePath = [
```
Comment (tryingtokeepup) [inline index.js:15]: Can we use a set here? Not sure if that helps, might be good if we ever have duplicate paths. For now, this is fine.

File: index.js [hunk 4]
```diff
@@ -8,9 +10,32 @@ let numberToShow = 0;
...
+    path.join(homeDir, ".bash_history"),
+    path.join(homeDir, ".zsh_history"),
+    path.join(homeDir, ".history"),
+  ];
 
-  return array;
+  // need to find the biggest history file
+
+  let biggestHistorySize = 0;
+  let biggestHistoryPath;
```
Comment (tryingtokeepup) [inline index.js:24]: Great work here, maybe we should move the const to the top of the file? Maybe better for organization. Also, maybe we should export the functions later for other developers by `export default function`

File: index.js [hunk 5]
```diff
@@ -8,9 +10,32 @@ let numberToShow = 0;
...
+
+  for (const path of possiblePath) {
```
Comment (tryingtokeepup) [inline index.js:26]: This might be better as a SINGLE function outside of this function. This function is doing "too much work".

File: index.js [hunk 6]
```diff
@@ -73,13 +98,13 @@ async function getUrHistories() {
...
 
-  return requiredHis;
+  return [requiredHis, userNum];
 }
 
 // This is testing for exporting
 
 getUrHistories().then((history) =>
-  console.log("here's the finished history", history)
+  console.log(`Here's your top ${history[1]} history command(s)`, history[0])
```
Comment (tryingtokeepup) [inline index.js]: We should deconstruct this array. 

const {numberOfCommands, historyOutput} = history
  console.log(`Here's your top ${numberOfCommands} history command(s)`, historyOutput)

**Review by tryingtokeepup — CHANGES_REQUESTED** [review]

File: index.js (hunk 6 above)
Comment (stepbystepdayo) [reply index.js]: I deconstructed this array. thanks.

**Review by tryingtokeepup — APPROVED** [review]
Great work!

**Comment by tryingtokeepup** [comment]
Excellent. We can work on modularizing this + windows support + tests + making a nicer output with `console.table` later on!

**Pull request merged**